django_react_proj/users-fe/src/components/NewUserForm/NewUserForm.js

```diff
@@ -10,13 +10,32 @@ const NewUserForm = ({ user, resetState, toggle }) => {
   const [name, setName] = useState(user?.name || '');
   const [password, setPassword] = useState(user?.password || '');
   const [deadline, setDeadline] = useState(user?.deadline || '');
+  const [taskType, setTaskType] = useState(user?.taskType || '');
+
+  const types = [
+    {
+      type: '',
+    },
+    {
+      type: 'Meeting',
+    },
+    {
+      type: 'Email',
+    },
+    {
+      type: 'Housework',
+    },
+  ];
 
   const onNameChange = (event) => {
     setName(event.target.value);
   };
   const onPasswordChange = (event) => {
     setPassword(event.target.value);
   };
+  const onTaskTypeChange = (event) => {
+    setTaskType(event.target.value);
+  };
   const defaultIfEmpty = (value) => (value === '' ? '' : value);
```

Comment:
I don't get it. It always returns value stored in variable `value`.

Reply:
So, it should be just 'const defaultIfEmpty = (value) => value;' ?

Reply:
just `value={deadline}` in `TimePicker` ond `value={raskType}` itd.